eo-maven-plugin/src/test/java/org/eolang/maven/optimization/OptCachedTest.java

```diff
@@ -50,73 +49,81 @@ final class OptCachedTest {
     /**
      * Test case for XML program in cache.
      *
-     * @param tmp Temp dir
+     * @param cache Temp cache dir.
+     * @param dir Temp program dir
      * @throws IOException if I/O fails
      * @todo #2422:60min returnsFromCacheIfXmlAlreadyInCache: this test is unstable.
      *  We should resolve issues with unstable failures and only
      *  then enable the test.
      *  Also, see this <a href="https://github.com/objectionary/eo/issues/2727">issue</a>.
      */
-    @Disabled
     @Test
-    void returnsFromCacheIfXmlAlreadyInCache(@TempDir final Path tmp) throws IOException {
-        final XML program = OptCachedTest.program(ZonedDateTime.now());
-        OptCachedTest.save(tmp, program);
+    void returnsFromCacheIfXmlAlreadyInCache(@TempDir final Path cache, @TempDir final Path dir)
+        throws Exception {
+        final XML xml = OptCachedTest.program();
+        final Path program = OptCachedTest.save(dir, xml, 0);
+        OptCachedTest.save(cache, xml, 0);
         MatcherAssert.assertThat(
             "We expected that the program will be returned from the cache.",
             new OptCached(
                 path -> {
                     throw new IllegalStateException("This code shouldn't be executed");
                 },
-                tmp
-            ).apply(program),
-            Matchers.equalTo(program)
+                cache,
+                program
+            ).apply(xml),
+            Matchers.equalTo(xml)
         );
     }
 
-    @Disabled
     @Test
-    void returnsFromCacheButTimesSaveAndExecuteDifferent(@TempDir final Path tmp)
-        throws IOException {
-        final XML program = OptCachedTest.program(ZonedDateTime.now().minusMinutes(2));
-        OptCachedTest.save(tmp, program);
+    void returnsFromCacheButTimesSaveAndExecuteDifferent(
```

Comment:
@Yanich96 What is "save time" and "execute time"? As I understand "save time" is a time when we saved a file, but what about "execute time"? Is it "last modified time"?